common/arrow/src/parquet_write.rs

```diff
@@ -45,6 +45,7 @@ where
     for group in row_groups {
         file_writer.write(group?)?;
     }
-    let (size, file_meta_data) = file_writer.end_ext(None)?;
-    Ok((size, file_meta_data))
+    let file_size = file_writer.end(None)?;
+    let (_meta_size, thrift_file_meta_data) = file_writer.into_inner_and_metadata();
```

Comment:
```suggestion
    let (_, thrift_file_meta_data) = file_writer.into_inner_and_metadata();
```

Reply:
er... let's keep this. seems it is not harmful and informative...